src/main/java/seedu/address/ui/EateryCard.java

```diff
@@ -44,8 +44,8 @@ public EateryCard(Eatery eatery, int displayedIndex) {
         this.eatery = eatery;
         id.setText(displayedIndex + ". ");
         name.setText(eatery.getName().fullName);
-        category.setText(eatery.getCategory().getName());
         address.setText(eatery.getAddress().value);
+        category.setText(eatery.getCategory().getName());
```

Comment:
why u move

Reply:
move as in down? I followed the convention of category after address